README.md

```diff
@@ -0,0 +1,13 @@
+### Context
+
+This is a ruby application that mimics the game hangman.  
+When the game starts, it generates a random word.  The random word is masked before displaying on screen.  The game continues until the number of attempts is exhausted or the masked word is revealed by the user successfully guessing all masked characters.
+
+### The User stories
+As a player, I want to start a game of Hangman
+As a player, I want to see the characters I’ve guessed correctly in a word
+As a player, I want to see the characters I have guessed incorrectly
+As a player, I want to see how many lives I have left
+As a player, I want to see a victory message when I have won
+As a player, I want to see a message when I have lost, including the word that I didn’t guess
+As a player, I want to be able to play again with a different word so that I can play many times
```

Comment:
Consider using checkboxes to indicate which stories are complete. This can be achieved by inserting `* [ ] ` before each line (or `* [x] ` if it's complete).